Index: Findiriutils.cmake
===================================================================
--- Findiriutils.cmake	(revision 73)
+++ Findiriutils.cmake	(working copy)
@@ -1,20 +0,0 @@
-FIND_PATH(iriutils_INCLUDE_DIR exceptions.h mutex.h mutexexceptions.h event.h eventserver.h eventexceptions.h thread.h threadserver.h threadexceptions.h comm.h commexceptions.h log.h logexceptions.h /usr/include/iriutils /usr/local/include/iriutils)
-
-FIND_LIBRARY(iriutils_LIBRARY
-    NAMES iriutils
-    PATHS /usr/lib /usr/local/lib /usr/local/lib/iriutils) 
-
-IF (iriutils_INCLUDE_DIR AND iriutils_LIBRARY)
-   SET(iriutils_FOUND TRUE)
-ENDIF (iriutils_INCLUDE_DIR AND iriutils_LIBRARY)
-
-IF (iriutils_FOUND)
-   IF (NOT iriutils_FIND_QUIETLY)
-      MESSAGE(STATUS "Found iriutils: ${iriutils_LIBRARY}")
-   ENDIF (NOT iriutils_FIND_QUIETLY)
-ELSE (iriutils_FOUND)
-   IF (iriutils_FIND_REQUIRED)
-      MESSAGE(FATAL_ERROR "Could not find iriutils")
-   ENDIF (iriutils_FIND_REQUIRED)
-ENDIF (iriutils_FOUND)
-
Index: src/CMakeLists.txt
===================================================================
--- src/CMakeLists.txt	(revision 73)
+++ src/CMakeLists.txt	(working copy)
@@ -12,7 +12,11 @@
 ADD_LIBRARY(iriutils SHARED ${sources})
 
 #edit the following line to add the necessary system libraries (if any)
-TARGET_LINK_LIBRARIES(iriutils pthread rt)
+IF (APPLE)
+    TARGET_LINK_LIBRARIES(iriutils pthread)
+ELSE(APPLE)
+    TARGET_LINK_LIBRARIES(iriutils pthread rt)
+ENDIF(APPLE)
 
 INSTALL(TARGETS iriutils
   RUNTIME DESTINATION bin
@@ -22,6 +26,6 @@
 
 INSTALL(FILES ${headers} DESTINATION include/iriutils)
 
-INSTALL(FILES ../Findiriutils.cmake DESTINATION ${CMAKE_ROOT}/Modules/)
+INSTALL(FILES ../Findiriutils.cmake DESTINATION ${CMAKE_INSTALL_PREFIX}/Modules/)
 
 ADD_SUBDIRECTORY(examples)
Index: src/examples/CMakeLists.txt
===================================================================
--- src/examples/CMakeLists.txt	(revision 73)
+++ src/examples/CMakeLists.txt	(working copy)
@@ -32,7 +32,11 @@
 ADD_EXECUTABLE(test_time test_time.cpp)
 
 # edit the following line to add the necessary libraries
-TARGET_LINK_LIBRARIES(test_time iriutils pthread rt) 
+IF (APPLE)
+    TARGET_LINK_LIBRARIES(test_time iriutils pthread)
+ELSE(APPLE)
+    TARGET_LINK_LIBRARIES(test_time iriutils pthread rt)
+ENDIF(APPLE)
 
 # edit the following line to add the source code for the example and the name of the executable
 ADD_EXECUTABLE(test_logs test_logs.cpp)
Index: src/time/ctime.cpp
===================================================================
--- src/time/ctime.cpp	(revision 73)
+++ src/time/ctime.cpp	(working copy)
@@ -104,7 +104,14 @@
 
   if(milliseconds<0.0)
   {
-    clock_gettime(CLOCK_REALTIME, &time_temp );
+    #if _POSIX_TIMERS > 0
+      clock_gettime(CLOCK_REALTIME, &time_temp );
+    #else
+      struct timeval tv;
+      gettimeofday(&tv, NULL);
+      time_temp.tv_sec = tv.tv_sec;
+      time_temp.tv_nsec = tv.tv_usec*1000;
+    #endif
   }
   else
   {
Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 73)
+++ CMakeLists.txt	(working copy)
@@ -12,7 +12,9 @@
 
 SET(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/bin)
 SET(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/lib)
-SET(CMAKE_INSTALL_PREFIX /usr/local)
+IF(NOT DEFINED CMAKE_INSTALL_PREFIX)
+    SET(CMAKE_INSTALL_PREFIX /usr/local)
+ENDIF(NOT DEFINED CMAKE_INSTALL_PREFIX)
 
 IF (NOT CMAKE_BUILD_TYPE)
  SET(CMAKE_BUILD_TYPE "DEBUG") 
@@ -21,7 +23,7 @@
 SET(CMAKE_C_FLAGS_DEBUG "-g -Wall -D_REENTRANT")
 SET(CMAKE_C_FLAGS_RELEASE "-O3 -D_REENTRANT")
 
-INSTALL(FILES  iri.conf DESTINATION /etc/ld.so.conf.d/)
+# INSTALL(FILES  iri.conf DESTINATION /etc/ld.so.conf.d/)
 
 ADD_SUBDIRECTORY(src)
 
