Index: src/CMakeLists.txt
===================================================================
--- src/CMakeLists.txt	(revision 42)
+++ src/CMakeLists.txt	(working copy)
@@ -69,6 +69,6 @@
 
 INSTALL(FILES ${headers} DESTINATION include/iriutils)
 
-INSTALL(FILES ../Findcomm.cmake DESTINATION ${CMAKE_ROOT}/Modules/)
+INSTALL(FILES ../Findcomm.cmake DESTINATION ${CMAKE_INSTALL_PREFIX}/Modules/)
 
 ADD_SUBDIRECTORY(examples)
Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 42)
+++ CMakeLists.txt	(working copy)
@@ -11,7 +11,9 @@
 
 SET(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/bin)
 SET(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/lib)
-SET(CMAKE_INSTALL_PREFIX /usr/local)
+IF(NOT DEFINED CMAKE_INSTALL_PREFIX)
+    SET(CMAKE_INSTALL_PREFIX /usr/local)
+ENDIF(NOT DEFINED CMAKE_INSTALL_PREFIX)
 
 IF (NOT CMAKE_BUILD_TYPE)
  SET(CMAKE_BUILD_TYPE "DEBUG") 
