Index: src/CMakeLists.txt
===================================================================
--- src/CMakeLists.txt	(revision 33)
+++ src/CMakeLists.txt	(working copy)
@@ -4,8 +4,12 @@
 SET(headers segway_rmp200.h segway_rmp200_exceptions.h)
 
 # edit the following line to find the necessary packages
-FIND_PACKAGE(iriutils)
-FIND_PACKAGE(comm)
+IF(NOT DEFINED iriutils_FOUND OR NOT DEFINED iriutils_INCLUDE_DIR OR NOT DEFINED iriutils_LIBRARY)
+    FIND_PACKAGE(iriutils)
+ENDIF(NOT DEFINED iriutils_FOUND OR NOT DEFINED iriutils_INCLUDE_DIR OR NOT DEFINED iriutils_LIBRARY)
+IF(NOT DEFINED comm_FOUND OR NOT DEFINED comm_INCLUDE_DIR OR NOT DEFINED comm_LIBRARY)
+    FIND_PACKAGE(comm)
+ENDIF(NOT DEFINED comm_FOUND OR NOT DEFINED comm_INCLUDE_DIR OR NOT DEFINED comm_LIBRARY)
 
 # edit the following line to add the necessary include directories
 INCLUDE_DIRECTORIES(.)
@@ -34,6 +38,6 @@
 
 INSTALL(FILES ${headers} DESTINATION include/iridrivers)
 
-INSTALL(FILES ../Findsegway_rmp_200.cmake DESTINATION ${CMAKE_ROOT}/Modules/)
+INSTALL(FILES ../Findsegway_rmp_200.cmake DESTINATION ${CMAKE_INSTALL_PREFIX}/Modules/)
 	
 ADD_SUBDIRECTORY(examples)
Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 33)
+++ CMakeLists.txt	(working copy)
@@ -12,8 +12,11 @@
 
 SET(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/bin)
 SET(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/lib)
-SET(CMAKE_INSTALL_PREFIX /usr/local)
 
+IF(NOT CMAKE_INSTALL_PREFIX)
+  SET(CMAKE_INSTALL_PREFIX /usr/local)
+ENDIF(NOT CMAKE_INSTALL_PREFIX)
+
 IF (NOT CMAKE_BUILD_TYPE)
  SET(CMAKE_BUILD_TYPE "DEBUG") 
 ENDIF (NOT CMAKE_BUILD_TYPE)
