Index: src/CMakeLists.txt
===================================================================
--- src/CMakeLists.txt	(revision 26)
+++ src/CMakeLists.txt	(working copy)
@@ -23,6 +23,6 @@
 
 INSTALL(FILES ${headers} DESTINATION include/iridrivers)
 
-INSTALL(FILES ../Findsegway_rmp200.cmake DESTINATION ${CMAKE_ROOT}/Modules/)
+INSTALL(FILES ../Findsegway_rmp200.cmake DESTINATION ${CMAKE_INSTALL_PREFIX}/Modules/)
 	
 ADD_SUBDIRECTORY(examples)
Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 26)
+++ CMakeLists.txt	(working copy)
@@ -12,7 +12,9 @@
 
 SET(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/bin)
 SET(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/lib)
-SET(CMAKE_INSTALL_PREFIX /usr/local)
+IF(NOT DEFINED CMAKE_INSTALL_PREFIX)
+    SET(CMAKE_INSTALL_PREFIX /usr/local)
+ENDIF(NOT DEFINED CMAKE_INSTALL_PREFIX)
 
 IF (NOT CMAKE_BUILD_TYPE)
  SET(CMAKE_BUILD_TYPE "DEBUG") 
